pages/common/sqsc.md

```diff
@@ -0,0 +1,31 @@
+# sqsc
+
+> A command line AWS SQS client.
+
+- List all queues:
+
+`sqsc lq [queue-prefix]`
+
+- List all messages in a queue:
+
+`sqsc ls <queue-name>`
+
+- Copy all messages from one queue to another:
+
+`sqsc cp <from-queue-name> <to-queue-name>`
+
+- Move all messages from one queue to another:
+
+`sqsc mv <from-queue-name> <to-queue-name>`
+
+- Descibe a queue:
```

Comment:
Typo - `Descibe` -> `Describe`?

Also, what do you mean by `describe`?

Reply:
Updated.  Describes a queue's attributes

<img width="484" alt="image" src="https://user-images.githubusercontent.com/1824221/36164169-ceffa746-10a8-11e8-986c-0738babfafe1.png">